compliance/compliance.go

```diff
@@ -1925,3 +1930,44 @@ func AddToNonexistentNetworkInstance(c *fluent.GRIBIClient, t testing.TB, _ ...T
 			AsResult(),
 		chk.IgnoreOperationID())
 }
+
+// Tests a case where two nexthops with duplicate contents are temporarily programmed
+func AddDuplicateNexthop(c *fluent.GRIBIClient, t testing.TB, _ ...TestOpt) {
```

Comment:
Optionally, you use the pattern used [here](https://github.com/openconfig/gribigo/blob/main/compliance/compliance.go#L1600-L1627) where you have a `wantACK` argument and use `makeTestWithACK` to be able to have this be tested against RIB and FIB ACKs (if there is ever a case where RIB ACK is also interesting here).

Reply:
I don't think we care about that at this point - if we ever do we can add it in.